pyke/kepsff.py

```diff
@@ -0,0 +1,760 @@
+    these algorithms, as coded, are not perfect. We recommend and welcome
+    tinkering and further development by the community in order to make these
+    tools better. This case is also precisely what the K2 community requires
+    in the sense that the archive users, here Andrew Vanderburg and John
+    Johnson, are seeking solutions to their own data reduction needs, becoming
+    self-sufficient, and reporting their findings and methods to us all.
+
+    Preparation work is required before kepsff can provide a good correction.
+    First, a light curve needs to be extracted a from Target Pixel File
+    (kepextract) using a user-customized pixel aperture defined within kepmask.
+    The resulting light curve also needs to be detrended, i.e. the astrophysics
+    needs to be removed as much as possible (kepflatten) in order for the
+    correction to be as precise as possible. Note that the astrophysics is not
+    removed permanently, but care has to be taken to prepare the data this way
+    before confidence in the correction can be built for individual light
+    curves on a case-by-case basis. A walkthrough example (Example E), starting
+    from a Target Pixel File and ending at a light curve mitigated for motion
+    systematics, is provided.
+
+    Parameters
+    ----------
+    nfile : str
+        The name of an input light curve FITS file. This file needs to contain
+        a minimum set of information: Mid-exposure times, cadence numbers,
+        simple-aperture flux, moment-derived and/or PSF-derived target
+        centroids, quality flags and Kepler standard keywords. All of these
+        properties are provided in archived light curves from the Kepler
+        mission, but need to be calculated for the K2 mission. The PyKE tool
+        kepextract calculates all these properties and provides an input file
+        suitable for kepsff. In addition, the input file also requires a
+        column of detrended data, where long-term astrophysics and
+        orbit-related systematics have been removed. The PyKE tool kepflatten
+        performs this pre-step, producing a FITS column of detrended data (at
+        low-frequencies) called DETSAP_FLUX.
+    outfile : str
+        The name of the output light curve FITS file. This file has the same
+        structure as the input file with three content changes. The aperture
+        photometry, DETSAP_FLUX, and detrended photometry, DETSAP_FLUX are
+        corrected for motion-correlated noise. The quality flag column,
+        SAP_QUALITY, includes a new bit-flag of value 131072 that identifies
+        exposures collected during on-board thruster firings, typically
+        occurring on a 6-hr cadence.
+    datacol : str
+        The name of the FITS data column containing detrended flux data (i.e.
+        low-frequency structure removed). If the input file is a product of
+        kepextract then this column is named DETSAP_FLUX.
```

Comment:
fixed in [ca4c2287](https://github.com/KeplerGO/PyKE/commit/ca4c22877d353be358880f78ed68f1e5d5dab7de)